integrationtests/tests/tests.rs

```diff
@@ -276,7 +276,7 @@ async fn receive_lightning_payment_valid_preimage() {
         .buy_preimage_offer(invoice.payment_hash(), &payment_amount, rng())
         .await
         .unwrap();
-    fed.run_consensus_epochs(4).await; // 2 epochs to process contract, 2 for preimage decryption
+    fed.run_consensus_epochs(4).await; // 1 epoch to process contract, 3 for preimage decryption
```

Comment:
Are you sure about that? Sounds more like a remnant of #171 and should be 1+1 now

Reply:
There's something goofy happening. Decryption shares show up every epoch after an offer has been purchased. https://github.com/fedimint/minimint/issues/179

Reply:
Fixed by https://github.com/fedimint/minimint/pull/180#issue-1284637114

Reply:
And I removed this comment so this PR should be good to go if there aren't any other problems.